doc/man3/OSSL_PROVIDER.pod

```diff
@@ -161,6 +163,12 @@ B<OSSL_PROVIDER_get_params()> described above.
 The parameters are returned by reference, not as copies, and so the elements of
 the I<param> array must have B<OSSL_PARAM_UTF8_PTR> as their B<data_type>.
 
+OSSL_PROVIDER_conf_get_bool() parses the global configuration parameter B<name>
+associated with provider B<prov> as a bolean value, returning a default value
+B<defval> when unable to retrieve or parse the parameter.
```

Comment:
```suggestion
I<defval> when unable to retrieve or parse the parameter.
```

Reply:
And there are other cases of `B<argument>` which should be `I<argument>` instead above.

Reply:
Done. Thanks.